src/extension/extension.ts

```diff
@@ -341,26 +342,6 @@ export class RunmeExtension {
       ),
     )
 
-    if (isPlatformAuthEnabled()) {
-      context.subscriptions.push(new StatefulAuthProvider(context, uriHandler))
-      const session = await getPlatformAuthSession(true)
-      if (session) {
-        const openDashboardStr = 'Open Dashboard'
-        const answer = await window.showInformationMessage(
```

Comment:
This `await` actually turns out to be permanently blocking until users respond to a non-modal notification. That's bad!

Can we fix this code to use the promise's then (as per this PR) to unblock completing initialization in a high priority separate PR @pastuxso? A user's action can't ever block the init method!